templates/common/_base/files/sshd-disable-ssh-key-dir.yaml

```
@@ -3,4 +3,4 @@ path: "/etc/ssh/sshd_config.d/10-disable-ssh-key-dir.conf"
 contents:
   inline: |
     # disable key lookup from ~/.ssh/authorized_keys.d/ on FCOS
-    AuthorizedKeysCommand none
+    AuthorizedKeysCommand /bin/true
```

can you try explicitly setting it to nil/empty?

```suggestion
    AuthorizedKeysCommand
```

---

this should re-instate the default